packages/esm-patient-immunizations-app/src/immunizations/immunizations.resource.tsx

```diff
@@ -43,13 +43,35 @@ export async function getImmunizationsConceptSet(
   return result;
 }
 
+export function useImmunizationsConceptSet() {
+  const { immunizationsConfig } = useConfig();
+  const conceptSetSearchTerm = immunizationsConfig?.vaccinesConceptSet;
+  const [source, code] = conceptSetSearchTerm.split(':');
+
+  const conceptSetMappingUrl = `/ws/rest/v1/concept?source=${source}&code=${code}&v=full`;
+  const conceptSetUuidUrl = `/ws/rest/v1/concept/${conceptSetSearchTerm}?v=full`;
```

Comment:
This makes me think we need some abstraction in esm-config for dealing with this consistently. It would be great if implementers could specify concepts in config by UUID, mapping, or FSN.